tests/unit/services/user-idle-test.js

```diff
@@ -24,18 +24,17 @@ test('init sets up event listeners', function (assert) {
   service.get('userActivity').trigger(event);
 
   let stub = service.get('resetTimeout');
-  assert.ok(stub.calledOnce, 'resetTimeout was called');
+  assert.ok(stub.calledTwice, 'resetTimeout was called');
 });
 
 test('resetTimeout', function (assert) {
   assert.expect(5);
-  let event = 'foo';
+
```

Comment:
Why remove this?

Reply:
I added it!  I have since reverted since I don't like the leaky-state rule on singletons.